geometry_msgs/WrenchStamped bias_outside
---
bool success
